faq/faq.rst

```diff
@@ -107,7 +107,7 @@ How do I create database roles, functions, extensions etc in a Citus cluster?
 
 Certain commands, when run on the coordinator node, do not get propagated to the workers:
 
-* ``CREATE ROLE/USER``
+* ``CREATE USER``
```

Comment:
I think we should keep an item for ROLE in here, in case the user does not know enough about the. difference between role and user

Reply:
I don't think it is the document's responsibility to identify difference between role and user, as we are generally assuming the reader has enough knowledge on PostgreSQL already ?

Reply:
If we were to choose one of the two, I suggest we go with `ROLE` because 
(a) PostgreSQL docs state that

> CREATE USER is now an alias for CREATE ROLE

(b) The `CREATE ROLE` statement is in the SQL standard,  whereas the `CREATE USER` statement is a PostgreSQL extension